deploy.sh

```diff
@@ -45,16 +53,26 @@ if [ "$ACCOUNT_ID" = "" ]; then
  exit 1;
 fi
 
-printf "Deploying %s on NEAR_ENV=%s with ACCOUNT_ID=%s\n\n" "$contract_name" "$NEAR_ENV" "$ACCOUNT_ID"
+printf "Deploying %s on NEAR_ENV=%s with ACCOUNT_ID=%s (patch=%s)\n\n" "$contract_name" "$NEAR_ENV" "$ACCOUNT_ID" "$patch"
+
 
 ./build.sh
 
 if [ "$contract_name" = "fungible_proxy" ]; then
+  set -x
```

Comment:
Is this for debugging? Do you want to keep it in there?

Reply:
Yes, I plan to keep it here to show the commands that are run, so it's easier to understand what's going on, especially if something went wrong, but not only.